src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -412,9 +412,27 @@ class TcgStatics {
   static healAll(PokemonCardSet target, Source source=Source.ATTACK){
     bg().em().run(new RemoveDamageCounter(target,source,target.damage))
   }
+  //Leaving this in since legacy code probably depends on it.
   static reduceDamageNextTurn (HP reduce, Move thisMove){
```

Comment:
I think it's better to change the method name IMO to `reduceDamageToSelfNextTurn` ?

Reply:
The goal of this PR was to change ReduceDamageNextTurn so that it would be ignored by no effect damage. Ideally I would have removed this method entirely but I assumed some legacy code would depend on it. What would be the benefit of renaming this over deleting it?